test/ConsoleModule.test.cpp

```diff
@@ -15,6 +15,10 @@ TEST(ConsoleModuleLoader, ModuleWithoutDLL)
 
 	// Test will need temporary module directory with no DLL present
 	// Ensure module directory ends with a trailing slash
+
+	const auto opuDirectory = GetOpuDirectory();
+	fs::create_directory(opuDirectory); // create_directory throws if parent directory does not exist
+
 	const auto moduleDirectory = fs::path(GetOpuDirectory()) / moduleName;
 	fs::create_directory(moduleDirectory);
```

Comment:
Perhaps we should be using `std::filesystem::create_directories` instead:
https://en.cppreference.com/w/cpp/filesystem/create_directory
> 3) Executes (1) for every element of p that does not already exist. If p already exists, the function does nothing (this condition is not treated as an error).

Reply:
Just tested and create_directories vice create_directory worked to recursively create child directories. I should have read the documentation closer... Good call. I didn't realize create_directories existed.